kqueen/engines/eks.py

```diff
@@ -202,13 +246,15 @@ def cluster_list(self):
     @classmethod
     def engine_status(cls, **kwargs):
         try:
-            aws_access_key = kwargs.get('aws_access_key', '')
-            aws_secret_key = kwargs.get('aws_secret_key', '')
+            aws_access_key = kwargs['aws_access_key']
+            aws_secret_key = kwargs['aws_secret_key']
+            aws_region = kwargs['aws_region']
         except Exception:
```

Comment:
We expected KeyError here, I think we should except that type of exception and do not log it, just log message, that credentials are missed